### DISCUSSION TITLE
Show mistyped character in error color fading out

### DISCUSSION THREAD

**Discussion by sigmondkukla** [monkeytypegame/monkeytype#1998] [Ideas] [answered]
Often when I am practicing typing I make a mistake that gets me out of my "flow" or routine of typing. When this happens, it is often in the middle of a word. Monkeytype doesn't show the character that you mistyped, instead it just turns the correct character red (or the error color you chose).
It would be great if Monkeytype would highlight the mistyped character in red, and show what character you incorrectly typed as a fading overlay to allow you to identify and quickly correct your mistake.
Here is a screen recording from Typing Club (a typing learning site for middle-school age students) that demonstrates this:
![Screen recording 2021-10-12 2 46 40 PM](https://user-images.githubusercontent.com/50522829/137012514-c9eaa14a-6fbf-4b84-8c3f-f8a6e790df9d.gif)

**Comment by Miodec** [comment, marked as answer]
Try using the `indicate typos` setting. Im not sure about changing / covering the actual character as it could be confusing to what needs to be typed at that point.

**Reply by janniks** [reply]
Does `indicate typos` still work? What should it look like? It all just looks the same on my monketype

♥️🐒⌨️

**Reply by Ferotiq** [reply]
![image](https://user-images.githubusercontent.com/64989416/153040904-6c47bcaa-4f42-43e9-9c2a-06435a9aa087.png)
This is what it should look like.
It appears under the word.

**Reply by janniks** [reply]
Ah thanks, I had `stop on letter` enabled. That option disables `indicate typos` — I was hoping to customize visually to be similar to keybr (which stops on letter but highlights the letter red, even after its typed correctly)

**Reply by Miodec** [reply]
We could add another type of indicate typos where it overlays the letter after giving it some more thought. When i initially responded i think in my mind the letter would be replaced instead of fading out.

**Reply by janniks** [reply]
Good idea 👍🏻 

I'll also think about it a bit more and maybe open my first PR here — just for the Discord badge 😈

**Reply by Miodec** [reply]
Added #2444
